ci-operator/step-registry/hypershift/mce/aws/create/hypershift-mce-aws-create-chain.yaml

```diff
@@ -45,6 +45,17 @@ chain:
           oc logs -n hypershift -lapp=operator --tail=-1 -c operator | grep -v "info" > $ARTIFACT_DIR/hypershift-errorlog.txt
         }
 
+        support_n-2() {
```

Comment:
It appears not good to use '-' in a function name? Though it may work sometimes, it may be not always compatible.
```
$ cat test.sh
support_n-2() {
  echo "Funcation name can contain '-'"
}

support_n-2

$ bash test.sh
Funcation name can contain '-'

$ sh test.sh
test.sh: line 3: `support_n-2': not a valid identifier
```
Google search seems to show discussion about it too. So we'd better avoid '-' in function names?